packages/node-sass-once-importer/src/index.ts

```diff
@@ -2,43 +2,69 @@ import {
   buildIncludePaths,
   resolveUrl,
 } from 'node-sass-magic-importer/dist/toolbox';
+import { ImporterReturnType, AsyncImporter } from 'node-sass';
+import * as md5 from 'nodejs-md5';
 
 const EMPTY_IMPORT = {
   file: ``,
   contents: ``,
 };
 
-export = function onceImporter() {
-  return function importer(url: string, prev: string) {
+interface NodeSassOnceImporterContext {
+  store: Set<string>
+}
+export = function onceImporter(): AsyncImporter {
+  let nodeSassOnceImporterContext: NodeSassOnceImporterContext;
+
+  return function importer(url: string, prev: string, done: (data: ImporterReturnType) => void) {
     const nodeSassOptions = this.options;
     // Create a context for the current importer run.
     // An importer run is different from an importer instance,
     // one importer instance can spawn infinite importer runs.
-    if (!this.nodeSassOnceImporterContext) {
-      this.nodeSassOnceImporterContext = {
+    if (!nodeSassOnceImporterContext) {
+      nodeSassOnceImporterContext = {
```

Comment:
It has been some years but if I remember correctly, using `this` is/was necessary here because of something node-sass specific. But not 100% sure if I remember correctly and if this is still relevant today. As long as tests pass, it should be ok. But currently they don't. But I didn't investigate why, so maybe completely unrelated.